workspacesView.js

```diff
@@ -16,7 +16,9 @@ const Util = Self.imports.util;
 
 function override() {
     global.vertical_overview.GSFunctions['WorkspacesView'] = Util.overrideProto(WorkspacesView.WorkspacesView.prototype, WorkspacesViewOverride);
-    log('You may see an error below,\nSecondaryMonitorDisplay is defined as const for some reason.\nSecondaryMonitorDisplay = ' + WorkspacesView.SecondaryMonitorDisplay + '\nSince I\'m overriding values in that const an error show might show up.\n Feel free to ignore it');
+    log('You may see an error below,\nSecondaryMonitorDisplay is defined as const for some reason\nSince I\'m overriding values in that const an error show might show up.\n Feel free to ignore it');
```

Comment:
>`an error show might show up`

Might as well fix that typo while you're at it, right?

Reply:
What typo?. Feel free to just push a commit to this that fixes that

Reply:
Oh, good eye! I had to re-read that 3 times to see the error lol